engines/libaio.c

```diff
@@ -123,7 +127,13 @@ static int fio_libaio_prep(struct thread_data *td, struct io_u *io_u)
 		if (o->nowait)
 			iocb->aio_rw_flags |= RWF_NOWAIT;
 	} else if (io_u->ddir == DDIR_WRITE) {
-		io_prep_pwrite(iocb, f->fd, io_u->xfer_buf, io_u->xfer_buflen, io_u->offset);
+		if ((td->o.zone_mode == ZONE_MODE_ZBD || td->o.zone_mode == ZONE_MODE_STRIDED) &&
```

Comment:
You initially mentioned that zoneappend option is valid only for zonemode=zbd.  So why adding strided here ? You will not have any zone information with strided mode. Or I am missing something here ?

Reply:
Yeah, strided mode is not needed for now. This will be removed in the v2 patches.
We can check for the strided mode support later.

Reply:
You are missing my point: zonemode=zbd supports zones, zonemode=strided does not. There will be no zone information with it, so no possibility of doing zone append.

Reply:
ok. this check will be removed for strided zone mode.